demisto_sdk/commands/format/update_integration.py

```diff
@@ -84,7 +84,12 @@ def set_fetch_params_in_config(self):
         correct fields.
         """
         if self.data.get('script', {}).get('isfetch') is True:
-            params = [_key for _key in self.data.get('configuration', [])]
+            # Creates a deep copy of the feed integration configuration so the 'defaultvalue` field would not get
+            # popped from the original configuration params.
+            params = [dict(config) for config in self.data.get('configuration', [])]
+            for param in enumerate(params):
```

Comment:
```suggestion
            for param in params:
```

Reply:
tests should have failed here, as param is a tuple instead of dict.
please make improve the tests.